andela/bp-esa-frontend
# 167486117-Make the app dashboard mobile responsive.

#### What does this PR do?
Makes the dashboard mobile responsive.

#### Description of Task to be completed?
The Application is not responsive when used on small devices, so this task enables the app to be usable on mobile devices as well.

#### How should this be manually tested?
- Load app in any web browser.
- In the browser dev tools enable the responsive mode so that the dashboard can be tested across different mobile browsers.
- Select different mobile devices to check the responsive nature of the app on mobile applications.



#### Any background context you want to provide?
None


#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167486117

#### Screenshots (If applicable)
<img width="605" alt="Screenshot 2019-09-12 at 16 34 25" src="https://user-images.githubusercontent.com/12638091/64788577-67c26e80-d57b-11e9-9c9d-6a9d800d3661.png">
<img width="527" alt="Screenshot 2019-09-12 at 16 35 00" src="https://user-images.githubusercontent.com/12638091/64788584-6b55f580-d57b-11e9-801d-7b76ad1ee7a8.png">
<img width="398" alt="Screenshot 2019-09-12 at 17 57 48" src="https://user-images.githubusercontent.com/12638091/64795598-06a09800-d587-11e9-8bc8-a40f7b1c0c05.png">
<img width="436" alt="Screenshot 2019-09-12 at 17 58 34" src="https://user-images.githubusercontent.com/12638091/64795608-0dc7a600-d587-11e9-8a72-f35d24304df9.png">
